_cp/instructions/seo.twig

```diff
@@ -0,0 +1,5 @@
+{# _cp/instructions/seo #}
+
+{{ "The following information will appear on social media sharing cards &amp; search engine results pages if no other content is available."|md }}
```

Comment:
Do we need to manually encode the ampersand into `&amp;`? Won’t Twig do it for us automatically if we just write `&`?